docs/index.md

```diff
@@ -0,0 +1,97 @@
+Documentation
+=============
+
+## Build the Label Sheet
+
+```php
+$labelSheet = new amattu2\LabelSheet('Avery5160');
+```
+
+### Add Text Labels
+
+Use the `addTextLabel` method to add a text label to the template. The method accepts an array of strings, an optional alignment parameter, and optional row and column parameters. This would typically be used for address labels.
+
+```php
+addTextLabel(array $lines, ?string $align = "C", int $row = null, int $col = null): void
+```
+
+```php
+$labelSheet->addTextLabel([
+  "line 1",
+  "line 2",
+  "line 3",
+  // ...
+])
+```
+
+### Add Image Labels
+
+Use the `addImageLabel` method to add an image label to the template. The method accepts a path to an image file and optional row and column parameters.
+
+```php
+addImageLabel(string $path, int $row = null, int $col = null): void
+```
+
+```php
+$labelSheet->addImageLabel('https://api.placeholder.app/image/350x350/.png');
+```
+
+### Add Custom Labels
+
+The `addCustomLabel` method allows you to expand upon the current label types (e.g. adding barcodes).
+Start by creating your custom label class.
+
+```php
+class ExampleLabel implements amattu2\LabelInterface
+{
+    private $data;
+
+    public function __construct($data)
+    {
+        $this->data = $data;
+    }
+
+    public function render($pdf, array $dimensions): bool
+    {
+        [$x, $y, $t] = $dimensions;
+        [$w, $h, $lines] = [$t['column_width'], $t['row_height'], $t['max_lines']];
+
+        $pdf->MultiCell($w, $h / $lines, $this->data, 0, 'R');
+
+        return true;
+    }
+
+    public function GetRow(): ?int
+    {
+        return null;
+    }
+
+    public function GetCol(): ?int
+    {
+        return null;
+    }
```

Comment:
These may need to be updated, depending on the outcome of #13. I've left them in their current form, so that this PR isn't dependent on that one.